pkg/storage/utils/decomposedfs/lookup/lookup.go

```diff
@@ -27,17 +27,18 @@ import (
 
 	user "github.com/cs3org/go-cs3apis/cs3/identity/user/v1beta1"
 	provider "github.com/cs3org/go-cs3apis/cs3/storage/provider/v1beta1"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"github.com/rogpeppe/go-internal/lockedfile"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/trace"
+
 	"github.com/cs3org/reva/v2/pkg/appctx"
 	"github.com/cs3org/reva/v2/pkg/errtypes"
 	"github.com/cs3org/reva/v2/pkg/storage/utils/decomposedfs/metadata"
 	"github.com/cs3org/reva/v2/pkg/storage/utils/decomposedfs/metadata/prefixes"
 	"github.com/cs3org/reva/v2/pkg/storage/utils/decomposedfs/node"
 	"github.com/cs3org/reva/v2/pkg/storage/utils/decomposedfs/options"
-	"github.com/google/uuid"
```

Comment:
1. Why do you sort it differently?
2. Why do you sort it in a wrong way?

Reply:
it is not a *wrong* way: https://github.com/golang/go/issues/32049

I don't know who introduced the grouping by std, local, external imports, but I wanted to play nice end enabled it in my VS Code. 

I'm happy to revert to stock go fmt ... but expect annoyed PR reviews by me in the future, then.